src/main/java/Duke.java

```diff
@@ -1,10 +1,58 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class Duke is the main class for the execution of Duke chatbot.
+ *
+ * @version 21 Jan 2021.
+ * @author Zhang Peng.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    /**
+     * This is the main method for the Duke class .
+     * @param args Unused.
+     * @return Nothing.
+     */
+
+    public static void main(String[] args) throws FileNotFoundException {
+        String input;
+        ArrayList<Task> arrayList = new ArrayList<>();
+
+        String path = "duke.txt";
+        File dukeFile = new File("duke.txt"); // doest matter if the path exit
+        boolean result;
+
+        try {
+            result = dukeFile.createNewFile();
+            if (result) {
+               System.out.println(".txt file created!");
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+
+        //read from the file content into the arrayList at the start.
+        Scanner s = new Scanner(dukeFile);
+        new Storage().loadingFile(arrayList, s);
+
+        System.out.println("--------------------------");
```

Comment:
I think it is better to store the horizontal lines as a constant as a final variable.

Reply:
got it. thank you.